packages/@okta/vuepress-site/docs/reference/api/features/index.md

```diff
@@ -0,0 +1,696 @@
+      }
+    }
+  }
+]
+```
+
+##### Error Response
+
+An invalid `featureId` returns a `404 Not Found` status code.
+
+```http
+HTTP/1.1 404 Not Found
+Content-Type: application/json
+
+{
+    "errorCode": "E0000007",
+    "errorSummary": "Not found: Resource not found: invalidFeatureId (Feature)",
+    "errorLink": "E0000007",
+    "errorId": "oaenAlD87mGTuWeGi7AjmhIBA",
+    "errorCauses": []
+}
+```
+
+## Features API Objects
+
+This API has the following objects:
+
+* [Feature](#feature-object)
+  * [Links](#links-object)
+* [Stage](#stage-object)
+
+### Feature Object
+
+#### Feature Properties
+
+The Feature model defines several properties:
+
+| Property      | Type                                                           | Description                                                           |
+| ------------- | -------------------------------------------------------------- | --------------------------------------------------------------------- |
+| `id`          | String                                                         | Unique identifier for this Feature. (Read-only)                       |
+| `type`        | String (Enum)                                                  | Current type of Feature. Possible value: `self-service`.              |
+| `status`      | String (Enum)                                                  | Current status of the Feature. Possible values: `ENABLED`, `DISABLED` |
+| `name`        | String                                                         | Name of the Feature                                                   |
+| `description` | String                                                         | Brief description of the Feature and what it provides                 |
+| `stage`       | [Stage Object](#stage-object)                                  | Current [Stage](#stage-object) for this Feature                       |
+| `_links`      | [JSON HAL](http://tools.ietf.org/html/draft-kelly-json-hal-06) | [Link relations](#links-object) for the Feature's current `status`    |
+
+### Feature Example
+
+```json
+{
+  "id": "ftrlBDFcGwYP2epXCGYn",
+  "type": "self-service",
+  "status": "ENABLED",
+  "name": "Example feature name",
+  "description": "Example feature description.",
+  "stage": {
+    "value": "EA"
+  },
+  "_links": {
+    "helpDoc": {
+      "href": "https://developer.okta.com/docs/concepts/feature-name/"
+    },
+    "self": {
+      "href": "https://{yourOktaDomain}/api/v1/features/ftrlBDFcGwYP2epXCGYn"
+    },
+    "dependents": {
+      "href": "https://{yourOktaDomain}/api/v1/features/ftrlBDFcGwYP2epXCGYn/dependents"
+    },
+    "dependencies": {
+      "href": "https://{yourOktaDomain}/api/v1/features/ftrlBDFcGwYP2epXCGYn/dependencies"
+    },
+    "disable": {
+      "href": "https://{yourOktaDomain}/api/v1/features/ftrlBDFcGwYP2epXCGYn/disable",
+        "hints": {
+          "allow": [
+            "POST"
+          ]
+        }
+    }
+  }
+}
+```
+
+### Links Object
```

Comment:
I think the section on the Stage object should come before this, to follow the order of objects in the JSON payload.